Makefile

```diff
@@ -137,15 +137,17 @@ bin: $(GOCMDS) $(GOINSTR)
 regen:       ## Refresh autogenerated files and reformat code.
              ## Use CHECK=1 to only validate clean repo after run.
              ##
-regen:
+regen: $(ADDLICENSE)
```

Comment:
question: maybe I'm missing something but where did this come from? I don't see this defined anywhere.

Reply:
It's in Bingo. The dependency was already added with the Helm change. This is just expanding it to check the rest of the repo.

https://github.com/GoogleCloudPlatform/prometheus-engine/blob/main/.bingo/Variables.mk#L20-L24

Reply:
Thanks! Looks good!